.github/workflows/benchmark-self-hosted.yml

```diff
@@ -19,7 +19,16 @@ jobs:
       matrix:
         config_dataset_info: [
             # config             dataset       lookahead   img-extension  source  loader         max-res  share-intrinsics
-            [skydio_front_end,   skydio-501,   15,         jpg,           wget,   colmap-loader, 600,     true,],
+            [
+              skydio_front_end,
```

Comment:
nit: i think was a bit more readable before, but up to you

Reply:
it autoformatted, will change back, thanks for catching

Reply:
done